content/docs/get_started/login_requiring.md

```diff
@@ -0,0 +1,163 @@
+---
+$title: Create a login-requiring AMP page
+$order: 3
+---
+Some user interactions with a page, such as leaving a comment, could be conditioned by a login flow. A login flow can be implemented with AMP using the [amp-access](https://www.ampproject.org/docs/reference/components/amp-access) component combined with the [amp-form](https://www.ampproject.org/docs/reference/components/amp-form) component. 
+To see a sample implementation, visit the [comment section sample](https://ampbyexample.com/samples_templates/comment_section/) at [ampbyexample.com](https://ampbyexample.com).
+
+The [comment section sample](https://ampbyexample.com/samples_templates/comment_section/) combines `amp-access` and `amp-form` to realize a comment section which is enabled only when an user has logged in. In order to explain how this sample works, we will follow the set of actions that will be performed once you land on the page.
+
+## Login
+
+The first time you land on the page, you can see 2 comments and a login button. 
+
+<amp-img src="/static/img/login-button.png" alt="Login button" height="290" width="300"></amp-img>
+
+If you look for the login button in the code, you will find:
+
+[sourcecode:html]
+<span amp-access="NOT loggedIn" role="button" tabindex="0" amp-access-hide>
+          <h5>Please login to comment</h5>
+          <a on="tap:amp-access.login-sign-in" class="button button-primary comment-button">Login</a>  
```

Comment:
Let's not use buttons that are not buttons in our examples. Can you change it to use a `<button>` tag both here and in ABE?